terraform/deployments/govuk-publishing-platform/frontends_origins.tf

```diff
@@ -82,6 +83,13 @@ resource "aws_lb_target_group" "router" {
   health_check {
     path = "/"
   }
+  
+  tags = merge(
+    local.additional_tags,
+    {
+      name      = "frontend-origin-router-tg"
```

Comment:
based on my new #231, it should be just `router-tg`

Reply:
ok